Doc/c-api/structures.rst

```diff
@@ -292,7 +292,7 @@ definition with the same method name.
 
    :attr:`flags` can be ``0`` for write and read access or :c:macro:`READONLY` for
    read-only access.  Using :c:macro:`T_STRING` for :attr:`type` implies
-   :c:macro:`READONLY`.  Only :c:macro:`T_OBJECT` and :c:macro:`T_OBJECT_EX`
+   :c:macro:`READONLY` and T_STRING will always interpreted as UTF-8 encoded string.  Only :c:macro:`T_OBJECT` and :c:macro:`T_OBJECT_EX`
```

Comment:
Could you please include the `:c:macro:` directive as is done in the preceding lines? It would be also good to wrap the line around 80 columns, in order to be consistent with the rest of the file.